### PR TITLE
Added layout for desktop screen

### PR DESCRIPTION
**Pull request by NayanKaran**
- Modified index.html for desktop screen. 
- Added CSS styles for desktop screen.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -145,43 +145,51 @@ <h3 class="project-heading">Multi-Post Stories</h3>
...
+          </p>
+          <section class="contacts">
+            <h2 class="contacts-heading">LET'S CONNECT</h2>
+            <ul class="social-media-icons no-bullets">
+              <li>
+                <a href="https://github.com/NayanKaran">
+                  <img alt="github icon" src="./images/icons/github_icon.svg" />
+                </a>
+              </li>
+              <li>
```
Comment (AnselemOdims) [inline index.html:171]: - [x] Please kindly ensure that the spacing at the top of the headline section matches that of the design. At the moment, your spacing is more than what was specified in the original design.
- [x] Please also ensure that the background image is properly set as in the original design.

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @NayanKaran , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [x] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (NayanKaran) [reply index.html:171]: Hi @AnselemOdims,  I checked the margin and padding values they are as specified by the design. can you please clarify this?

Thanks for pointing out the background image. I am going to change it.

**Review by KayLemba — APPROVED** [review]
Hi @NayanKaran ,

Your project is complete! There is nothing else to say other than... it's time to merge it 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**